experimental/dds/tree2/src/feature-libraries/editable-tree/editableTreeTypes.ts

```diff
@@ -325,6 +326,16 @@ export interface EditableField
 	 */
 	insertNodes(index: number, newContent: ITreeCursor | ITreeCursor[]): void;
 
+	/**
+	 * Moves nodes from a source to destination.
```

Comment:
This doesn't take in a "source", so I think this comment could use rephrasing. 
```suggestion
	 * Moves nodes from this field to destination.
```

Reply:
updated comment